sky/server/constants.py

```diff
@@ -40,6 +40,8 @@
 # The path to the API request database.
 API_SERVER_REQUEST_DB_PATH = '~/.sky/api_server/requests.db'
 
+PER_WORKER_THREAD_POOL_MEMORY_GB = 8 / 1024  # 8MB
```

Comment:
Can we fix the naming between fastapi workers, threadpool threads, and skypilot executors? I guess this is "memory used by the threadpool for each fastapi worker" but it's hard to understand at a glance